libraries/AP_L1_Control/AP_L1_Control.h

```diff
@@ -124,6 +124,10 @@ class AP_L1_Control : public AP_Navigation {
 
     AP_Float _loiter_bank_limit;
 
+    // remember reached_loiter_target decision
+    uint32_t last_reached_loiter_target_ms;
+    Location last_center_WP;
```

Comment:
All the other class variables in L1 have the `_` prefix.

Reply:
changed